bind assumes _Unwind_GetIP is a function which is not necessarily
true. In some implementations of libunwind it's a macro.
This fixes the build on Alpine on armhf and armv7.

--- a/bind/bind-9.11.36/lib/isc/backtrace.c
+++ b/bind/bind-9.11.36/lib/isc/backtrace.c
@@ -81,8 +81,7 @@ isc_backtrace_gettrace(void **addrs, int
 	return (ISC_R_SUCCESS);
 }
 #elif defined(BACKTRACE_GCC)
-extern int _Unwind_Backtrace(void* fn, void* a);
-extern void* _Unwind_GetIP(void* ctx);
+#include <unwind.h>
 
 typedef struct {
 	void **result;
